packages/gateway/index.ts

```diff
@@ -1,54 +1,21 @@
-import { ApolloServer, gql } from 'apollo-server';
+import { ApolloServer } from 'apollo-server';
 import { ApolloGateway } from '@apollo/gateway';
-import * as console from 'console';
+import * as fs from 'fs';
+import * as path from 'path';
```

Comment:
For the below suggestion:
```suggestion
import * as path from 'path';
import { ApolloServerPluginLandingPageGraphQLPlayground } from 'apollo-server-core';
```